distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/CatalogBundle.java

```diff
@@ -195,11 +203,15 @@ public CatalogFramework getCatalogFramework() throws InterruptedException {
     }
 
     public void setFanout(boolean fanoutEnabled) throws IOException {
-        Map<String, Object> properties = adminConfig.getDdfConfigAdmin()
-                .getProperties(CATALOG_FRAMEWORK_PID);
-        if (properties == null) {
-            properties = new Hashtable<>();
+        Map<String, Object> properties = null;
+        try {
+            properties = Optional.ofNullable(adminConfig.getAdminConsoleService()
+                    .getProperties(CATALOG_FRAMEWORK_PID))
+                    .orElse(new Hashtable<>());
+        } catch (NotCompliantMBeanException e) {
+            //ignore
```

Comment:
I can't not throw that exception. It just gets thrown because of the fact that it is an mbean. The itests should not be using the mbean this way, they should be using it through the mbean server or maybe via the osgi service that I created, if enough operations exist. This would be a MASSIVE change to our itests, which are now tightly coupled to that implementation and is not within the scope of this change.